public/app/app.ts

```diff
@@ -206,24 +207,19 @@ export class GrafanaApp {
       const modalManager = new ModalManager();
       modalManager.init();
 
-      let preloadResults: PluginPreloadResult[] = [];
+      // Initialize plugin extensions
+      const extensionsRegistry = new ReactivePluginExtensionsRegistry();
+      extensionsRegistry.register({
+        pluginId: 'grafana',
+        extensionConfigs: getCoreExtensionConfigurations(),
+      });
 
       if (contextSrv.user.orgRole !== '') {
-        // Preload selected app plugins
-        preloadResults = await preloadPlugins(config.apps);
+        preloadPlugins(config.apps, extensionsRegistry);
```

Comment:
Not sure this is an issue but... does this have implications on plugins that currently use `preload: true` for reasons other than plugin extensions? e.g. Will there be a spinner whilst waiting for easystart to load? Would we see layout shift if easystart/cloud app use preload for rendering cloud usage banners when a users homepage is set to something other than the default?

Reply:
Yes, this is something we need to take into consideration.

Reply:
hm.. easystart should not really need to be preloaded. Do you know why it is?

Reply:
Isn't that because of the notification banner that we added a while back? It displays the licensing/quota information etc.

Reply:
(It's probably [cloud-home-app](https://github.com/grafana/hg-cloud-home-admin/blob/main/src/plugin.json#L61), [grafana-easystart-app is not preloaded](https://github.com/grafana/cloud-onboarding/blob/main/plugin/src/plugin.json).)